spec/rails_helper.rb

```diff
@@ -229,9 +229,10 @@ def filling_in_ssn_without_dashes(args)
         end
 
         browser_console_logs = page.driver.browser.logs.get(:browser)
-        if browser_console_logs.length > 0
+        if browser_console_logs.any?
```

Comment:
there were a bunch of repetitive intercom messages, this just makes it less noisy by only printing the message once instead of 30 times